Tensile/SolutionStructs.py

```diff
@@ -2013,6 +2013,7 @@ def assignProblemIndependentDerivedParameters(state):
   def setGlobalLoadVectorWidth(state, tc, totalElements, grvw):
     validDepthU = True
     numThreadGrvw = int(state["NumThreads"] * grvw)
+    #print(f"DEBU:KOJI:numThreadGrvw={numThreadGrvw}, totalElements={totalElements}, grvw={grvw}")
```

Comment:
Can clean up debug output if not needed anymore

Reply:
Sorry. I forgot to delete debug code...
I will delete it.

Reply:
Removed!